version.h not found

There's a cmake issue on the master branch : 
```
cd /home/hoarau/dev
git clone https://github.com/rpclib/rpclib
cd rpclib 
mkdir build ; mkdir install
cd build
cmake .. -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=../install -DRPCLIB_ENABLE_LOGGING=1
make 
make install
```
Results in : 
```
CMake Error at cmake_install.cmake:44 (file):
  file INSTALL cannot find "/home/hoarau/dev/rpclib/build/version.h".
```

This is because `version.h` is generated in `PROJECT_SOURCE_DIR` 
https://github.com/rpclib/rpclib/blob/master/CMakeLists.txt#L203:L205 but install expects it in `PROJECT_BINARY_DIR` https://github.com/rpclib/rpclib/blob/master/CMakeLists.txt#L296

Also, I think you should add `config.h` in `.gitignore`, but that's really not an issue

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -293,7 +293,7 @@ install(DIRECTORY include/
     PATTERN "*.hpp"
     PATTERN "*.inl"
     PATTERN "*.in" EXCLUDE)
-install(FILES ${PROJECT_BINARY_DIR}/version.h DESTINATION include/rpc)
+install(FILES ${PROJECT_SOURCE_DIR}/include/rpc/version.h DESTINATION include/rpc)
 
 if(RPCLIB_GENERATE_COMPDB)
     set(CMAKE_EXPORT_COMPILE_COMMANDS "ON") # for YCM

--- a/examples/mandelbrot/mandelbrot.h
+++ b/examples/mandelbrot/mandelbrot.h
@@ -3,7 +3,7 @@
 #ifndef MANDELBROT_H_I0UEF3KR
 #define MANDELBROT_H_I0UEF3KR
 
-#include "msgpack.hpp"
+#include "rpc/msgpack.hpp"
 #include <vector>
 
 struct pixel {

